PowerUp/app/src/main/java/powerup/systers/com/ui/StartActivity.java

```diff
@@ -0,0 +1,185 @@
+/**
+ * @desc brings user to map if previous session is being opened. Otherwise,
+ * a new user will be brought to the “Avatar Room” to customize avatar
+ * upon starting the app.
+ */
+
+package powerup.systers.com.ui;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.graphics.Color;
+import android.graphics.drawable.ColorDrawable;
+import android.os.Bundle;
+import android.os.Handler;
+import android.support.v7.app.AlertDialog;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import powerup.systers.com.R;
+import powerup.systers.com.data.DataSource;
+import powerup.systers.com.data.SessionHistory;
+import powerup.systers.com.minesweeper.MinesweeperSessionManager;
+import powerup.systers.com.sink_to_swim_game.SinkToSwimSessionManager;
+import powerup.systers.com.ui.avatarRoom.AvatarRoomActivity;
+import powerup.systers.com.ui.map_screen.MapActivity;
+import powerup.systers.com.utils.InjectionClass;
+import powerup.systers.com.vocab_match_game.VocabMatchSessionManager;
+
+public class StartActivity extends Activity {
+
+    @BindView(R.id.startButtonMain)
```

Comment:
@geekanamika , looks like, its mentioning here, that variable, "startButton", doesn't have explicit scoping (private/public), it will make private variables vulnerable, to get modified by external class. 
So, if you meant to keep "startButton", as private add private, else getter/setter for public member (don't leave dangling like this)